src/SeleniumTestCase.php

```diff
@@ -3,51 +3,65 @@
 namespace Modelizer\Selenium;
 
 use Modelizer\Selenium\Services\Application as Laravel;
+use Modelizer\Selenium\Services\InteractsWithPage as Interactions;
+use Modelizer\Selenium\Services\WaitsForElements;
+use Modelizer\Selenium\Services\WorksWithDatabases;
 use PHPUnit_Extensions_Selenium2TestCase;
 
 class SeleniumTestCase extends PHPUnit_Extensions_Selenium2TestCase
 {
-    use Laravel;
+    use Laravel,
+        Interactions,
+        WorksWithDatabases,
+        WaitsForElements;
+
+    /**
+     * @var string
+     */
+    protected $baseUrl;
 
     protected function setUp()
     {
         $this->setUpLaravel();
-
-        $this->setBrowserUrl(env('APP_URL', 'http://localhost/'));
+        $this->baseUrl = env('APP_URL', 'http://localhost/');
+        $this->setBrowserUrl($this->baseUrl);
         $this->setBrowser(env('DEFAULT_BROWSER', 'chrome'));
     }
 
-    protected function visit($path)
-    {
-        $this->url($path);
-
-        return $this;
-    }
-
-    protected function see($text, $tag = 'html')
+    public function setupPage()
     {
-        $this->assertContains($text, $this->byTag($tag)->text());
-
-        return $this;
+        if (!empty(getenv('SELENIUM_WIDTH')) && !empty(getenv('SELENIUM_HEIGHT')) &&
+            is_int(intval(getenv('SELENIUM_WIDTH'))) && is_int(intval(getenv('SELENIUM_HEIGHT')))) {
+            $this->prepareSession()->currentWindow()->size([
+                'width'  => intval(getenv('SELENIUM_WIDTH')),
+                'height' => intval(getenv('SELENIUM_HEIGHT')),
+            ]);
+        }
     }
 
-    protected function hold($seconds)
+    /**
+     * Force selenium to wait.
+     *
+     * @param int|float $seconds The number of seconds or partial seconds to wait
+     *
+     * @return $this
+     */
+    protected function wait($seconds = 1)
     {
-        sleep($seconds);
+        usleep($seconds * 1000000);
```

Comment:
usleep allows you to do things like partial seconds.  sleep will only take an integer to sleep for.  but if i only wanted to wait for 0.5 seconds so the page can finish the animation on the page before continuing instead of waiting a whole second, the test runs faster.